trace_decoder/src/decoding.rs

```diff
@@ -316,7 +316,7 @@ impl ProcessedBlockTrace {
             {
                 // If we are writing a zero, then we actually need to perform a delete.
                 match val == &ZERO_STORAGE_SLOT_VAL_RLPED {
-                    false => storage_trie.insert(slot, val.clone()),
+                    false => storage_trie.insert(slot, val.clone()).unwrap(),
```

Comment:
Is there a reason to `unrwap()` here vs `?` and propagating the error up to be handled correctly by whomever? As it stands this will panic as the error is unhandled

Reply:
not sure how you use this code, but here yes, it will make more sense with `?`